aptos-move/block-executor/src/task.rs

```diff
@@ -68,8 +68,16 @@ pub trait ExecutorTask: Sync {
     /// Create an instance of the transaction executor.
     fn init(args: Self::Argument) -> Self;
 
+    /// Execute one single transaction given the view of the current state as a BTreeMap,
```

Comment:
this is annoying, you mentioned that and I am not clear what's going on, apology, I need a bit more time around this

Reply:
Indeed, need to create a common view on the executor side to deprecate both VersionedView and StateViewCache, and then we can unify the interfaces (and BTree or Multi-version becomes implementation detail). Also in the follow-up draft: https://github.com/aptos-labs/aptos-core/commit/39f055092afc2b67afcd7990ba2bc6714b90682d#diff-c357acdc9a799c00c416262b0fd8e627e1f882f603e63d77625ae0f0fd9535fa